torch/_subclasses/meta_utils.py

```diff
@@ -617,11 +617,15 @@ def __call__(
         source=None,
         dynamic_dims=None,
         constraint_dims=None,
+        force_fresh=False,
     ):
         # TODO: zero tensors?  We appear to have eliminated them by
         # excluding complex for now
         from torch._subclasses.fake_tensor import FakeTensor
 
+        if force_fresh:
+            del self.tensor_memo[WeakIdRef(t)]
```

Comment:
@ezyang this might have weird implications for dynamic shapes, we should discuss. Will ping you.